### PR TITLE
Setting factory to null after destroying it, so that the next time th…

### LINKED ISSUE #88: Feature: ability to start the clients after calling the stop method
Right now, it is not possible to re-start a client in pvAccessJava after stopping it.
I am proposing a fix for that.

### PR DESCRIPTION
**Pull request by juanfem**
…e start() method is called, a new factory is created. This fixes #88

### CODE REVIEW

**Comment by shroffk** [comment]
Ok, this makes sense to me.
Should I go ahead and merge it?

File: pvAccessJava/src/org/epics/ca/ClientFactory.java [hunk 1]
```diff
@@ -148,7 +149,7 @@ public void destroy() {
                 context.destroy();
             } catch (CAException e) {
                 e.printStackTrace();
-            }
+        }
```
Comment (ralphlange) [inline pvAccessJava/src/org/epics/ca/ClientFactory.java:152]: Is this a tab vs. spaces issue?

Comment (juanfem) [reply pvAccessJava/src/org/epics/ca/ClientFactory.java]: Ops, I missed that. Reverting to the previous state.

**Comment by juanfem** [comment]
I've added checks before setting factories to null. They are different for each protocol, because one is handling the exceptions and the other doesn't.

**Comment by juanfem** [comment]
Thanks!

**Pull request merged**